python/lsst/ts/electrometer/controller.py

```diff
@@ -129,14 +128,20 @@ async def send_command(self, command, has_reply=False):
 
     async def connect(self):
         """Open connection to the electrometer."""
-        self.commander.open()
+        try:
+            self.commander.open()
+        except serial.SerialException:
+            pass
```

Comment:
Are you sure you want to not follow up on a SerialException here? I see that this CSC doesn't support a simulation mode so perhaps at least logging an error would be wise to not let this pass unnoticed?

Reply:
Added a log call there, my logic is that eventually I'll write a simulator for it so that I can simulate it for the unit tests, which is why I threw it away even though its not a great idea as you rightly pointed out. Maybe I can find a smarter way to do it without risking ignoring a SerialException in operation.